presto-raptor/src/test/java/com/facebook/presto/raptor/metadata/TestRaptorMetadata.java

```diff
@@ -157,6 +174,26 @@ public void testDropColumn()
         assertNull(metadata.getColumnHandles(SESSION, tableHandle).get("orderkey"));
     }
 
+    @Test
+    public void testAddColumnAfterDropColumn()
+    {
+        assertNull(metadata.getTableHandle(SESSION, DEFAULT_TEST_ORDERS));
+        metadata.createTable(SESSION, buildTable(ImmutableMap.of(), tableMetadataBuilder(DEFAULT_TEST_ORDERS)
+                        .column("orderkey", BIGINT)
+                        .column("price", BIGINT)),
+                false);
+        ConnectorTableHandle tableHandle = metadata.getTableHandle(SESSION, DEFAULT_TEST_ORDERS);
+        assertInstanceOf(tableHandle, RaptorTableHandle.class);
+
+        RaptorTableHandle raptorTableHandle = (RaptorTableHandle) tableHandle;
+        ColumnHandle lastColumn = metadata.getColumnHandles(SESSION, tableHandle).get("orderkey");
```

Comment:
This name seems wrong since it's not the last column. How about `dropColumn` or just `column`